wiki/Tournaments/MCLT/4K2025/en.md

```diff
@@ -0,0 +1,386 @@
+| Hikikomori ::{ flag=CL }:: | **5** | -1 | ::{ flag=CL }:: Horuszed | [#1](https://osu.ppy.sh/community/matches/117028217) |  |
+| Ogata Kanna ::{ flag=CL }:: | **5** | 2 | ::{ flag=CL }:: TheTrixdo_YT | [#1](https://osu.ppy.sh/community/matches/117029124) | [#1](https://www.twitch.tv/videos/2370163235) |
+| Anterex ::{ flag=CL }:: | **5** | 0 | ::{ flag=CL }:: [REG]Levels | [#1](https://osu.ppy.sh/community/matches/117031119) | [#1](https://www.twitch.tv/videos/2370163448) |
+
+Sunday, 16 February 2025:
+
+| Player A |  |  | Player B | Match link | VOD link |
+| --: | :-: | :-: | :-- | :-- | :-- |
+| yukataa ::{ flag=CL }:: | 4 | **5** | ::{ flag=CL }:: Jeanne Da Rin | [#1](https://osu.ppy.sh/community/matches/117038115) | [#1](https://www.twitch.tv/videos/2370616128) |
+| kuraaaFangirl ::{ flag=CL }:: | **5** | 1 | ::{ flag=CL }:: UQU | [#1](https://osu.ppy.sh/community/matches/117039635) | [#1](https://www.twitch.tv/videos/2370701940) |
+| Shiny Sylveon ::{ flag=CL }:: | **5** | 4 | ::{ flag=CL }:: NikitodeuS | [#1](https://osu.ppy.sh/community/matches/117041804) | [#1](https://www.twitch.tv/videos/2371105628) |
+
+### Qualifiers
+
+The final standings for the Qualifier stage can be found in the following [spreadsheet](https://docs.google.com/spreadsheets/d/1-msr4rlozBQP-jWMIwBHAXzVtFRbS0wY7UxbO_wpy7I).
+
+| Seed | Player | Average Rank[^qualifiers-seeding] | avg. score[^qualifiers-tiebreaker] |
+| :-: | :-- | --: | --: |
+| #1 | ::{ flag=CL }:: b3njvv2 | 2.4 | 988,095 |
+| #2 | ::{ flag=CL }:: Anterex | 2.6 | 988,978 |
+| #3 | ::{ flag=CL }:: kuraaaFangirl | 4.4 | 981.640 |
+| #4 | ::{ flag=CL }:: Hikikomori | 4.4 | 980,398 |
+| #5 | ::{ flag=CL }:: Ogata Kanna | 6.6 | 975.031 |
+| #6 | ::{ flag=CL }:: Shiny Sylveon | 8.6 | 970.156 |
+| #7 | ::{ flag=CL }:: yukataa | 8.6 | 968,687 |
+| #8 | ::{ flag=CL }:: Yor Fangirl | 8.6 | 968,247 |
+| #9 | ::{ flag=CL }:: Best Sanallite | 10.8 | 964,395 |
+| #10 | ::{ flag=CL }:: Jeanne Da Rin | 11.2 | 963,878 |
+| #11 | ::{ flag=CL }:: NikitodeuS | 12 | 958.823 |
+| #12 | ::{ flag=CL }:: TheTrixdo_YT | 13.4 | 956,299 |
+| #13 | ::{ flag=CL }:: Horuszed | 13.4 | 947,361 |
+| #14 | ::{ flag=CL }:: UQU | 13.6 | 952,576 |
+| #15 | ::{ flag=CL }:: [REG]Levels | 14.2 | 956,143 |
+| #16 | ::{ flag=CL }:: Wantaso | 14.6 | 951,591 |
+
+## Ruleset
+
+### General
+
+1. Only players with the Chilean flag visible on their osu! profile are eligible to enter the tournament.
+2. [Tournament staff members](/wiki/Tournaments/Official_support#staff) must not participate as players in the tournament.
+3. Tournament schedules are based on Chilean continental time (UTC-3).
+4. To participate in the tournament, player **must** join [this Discord server](https://discord.gg/aSTBgHBpSN) after registration.
+5. This tournament is a 1v1 double-elimination competition for the osu!mania 4K game mode.
+6. The score system used will be ScoreV2, and the version used will be stable, not osu!lazer.
```

Comment:
```suggestion
6. Beatmap scoring is based on ScoreV2.
```

I checked [`LGA`](https://osu.ppy.sh/wiki/en/Tournaments/LGA/2024),[`COE`](https://osu.ppy.sh/wiki/en/Tournaments/COE/COE_2024) rulebooks, they usually talking game version only in introduction, while `osu!lazer tournament` means tournament running on lazer。
(Test Open haven't rulebooks on osu!wiki)

will leave remain review for osu!wiki maintancer.